src/controllers/active.directors.controller.ts

```diff
@@ -162,9 +163,9 @@ async function beginUpdateJourney(req: Request, res: Response, session: Session,
     dateOfBirth: appointment.dateOfBirth?.year + "-" + appointment.dateOfBirth?.month?.toString().padStart(2, '0') + "-" + appointment.dateOfBirth?.day?.toString().padStart(2, '0'),
     appointedOn: appointment.appointedOn? appointment.appointedOn: appointment.appointedBefore,
     occupation: occupation,
-    nationality1: nationalities && nationalities?.length > 0? formatTitleCase(nationalities[0]): "",
-    nationality2: nationalities && nationalities?.length > 1? formatTitleCase(nationalities[1]): "",
-    nationality3: nationalities && nationalities?.length > 2? formatTitleCase(nationalities[2]): "",
+    nationality1: nationalities && nationalities?.length > 0? lowerCaseAllWordsExceptFirstLetters(nationalities[0]): "",
```

Comment:
You might be able to drop the ? from nationalities?.length to get test coverage up. It's checked by first condition so the ? is not needed. For lines 166 to 168.

Reply:
? removed, test coverage has increased :D